src/sdg/templates/core/logger/event.html

```diff
@@ -0,0 +1,37 @@
+{% load i18n %}
```

Comment:
Can be moved under admin folder since its only for admin use: `admin/core/logger/event.html`